### DISCUSSION TITLE
Install FastForward extension/addon

### DISCUSSION THREAD

**Discussion by joaonunatings** [qutebrowser/qutebrowser#6992] [Q&A] [answered]
Is it possible to install [FastForward](https://fastforward.team/) extension?
I know there are greasemonkey scripts which I successfully installed.

**Comment by The-Compiler** [comment, marked as answer]
Not directly, as it's a Chromium extension, which don't work directly in qutebrowser (see the FAQ in #30 on that). Someone would need to take their ruleset and find a way to do the same in qutebrowser (possibly as a greasemonkey script right now, and as a Python extension in the future).

**Reply by joaonunatings** [reply]
Ok, I'll try and get my head around a greasemonkey script that does the same (maybe try and make one myself). Thank you
